modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -32,6 +34,8 @@ export function ConfigurationContainer({
 		isLinkedObjectDefinition ||
 		isReadOnly;
 
+	const scheduleToggleRef = useRef<boolean>(false);
```

Comment:
Hi @larissacribeiro, could you explaim to me why this is necessary?

Reply:
we talked in person and I will research if its better to use a ref or just declare a let